test/adapter/json/key_case_test.rb

```diff
@@ -64,28 +62,33 @@ def test_key_transform_dashed
           mock_request(:dashed)
           assert_equal({
             blog: { id: 1, :"special-attribute" => 'neat', articles: nil }
-          }, @adapter.serializable_hash(@options))
+          }, adapter.serializable_hash)
         end
 
         def test_key_transform_unaltered
           mock_request(:unaltered)
           assert_equal({
             blog: { id: 1, special_attribute: 'neat', articles: nil }
-          }, @adapter.serializable_hash(@options))
+          }, adapter.serializable_hash)
         end
 
         def test_key_transform_camel
           mock_request(:camel)
           assert_equal({
             Blog: { Id: 1, SpecialAttribute: 'neat', Articles: nil }
-          }, @adapter.serializable_hash(@options))
+          }, adapter.serializable_hash)
         end
 
         def test_key_transform_camel_lower
           mock_request(:camel_lower)
           assert_equal({
             blog: { id: 1, specialAttribute: 'neat', articles: nil }
-          }, @adapter.serializable_hash(@options))
+          }, adapter.serializable_hash)
+        end
+
+        def adapter
```

Comment:
this should just be `serializable(@blog, serializer: CustomBlogSerializer, adapter: :json, options: whatever)`